igvm/settings.py

```diff
@@ -290,6 +293,17 @@
     }
 ]
 
+# The loadtested CPU usage thresholds per hypervisor hardware model,
+# before we are experiencing reasonable steal time.
+HYPERVISOR_CPU_THRESHOLDS = {
+    'Dell_M610': 50,  # untested
+    'Dell_M710': 50,  # untested
+    'Dell_M620': 60,
+    'Dell_M630': 70,
+    'Dell_M640': 75,
+    'Dell_R6515': 50,
```

Comment:
This seems rather low for a brand new hardware with a single NUMA node.

Reply:
Well, we loadtested the R6515 with overbooking (96), all CPUs incl. HT (64) and only pCPUs (32) and noticed reasonable stealtime at 50% of CPU load of the HV already. Same we did for the M6X0's, so the values are comparable.

Seems like AMDs HT is not capable of handling our workload properly.

Reply:
If that's the number found by experiment, then I trust it.